.github/workflows/check-and-release-main.yml

```diff
@@ -4,14 +4,14 @@ name: Check and Release
 on:
   push:
     branches:
-      - master
+    - master
```

Comment:
See also PR https://github.com/jenstroeger/python-package-template/pull/72. Should we merge this PR first, and then rebase https://github.com/jenstroeger/python-package-template/pull/72 on top?

Reply:
Yes, sounds good.

Reply:
Uhm?